Marlin/SanityCheck.h

```diff
@@ -344,14 +346,8 @@
 /**
  * Single Stepper Dual Extruder with switching servo
  */
-#if ENABLED(SWITCHING_EXTRUDER)
-  #if ENABLED(DUAL_X_CARRIAGE)
-    #error "SWITCHING_EXTRUDER and DUAL_X_CARRIAGE are incompatible."
-  #elif (EXTRUDERS != 2) && (EXTRUDERS != 4)
-    #error "SWITCHING_EXTRUDER requires exactly 2 or 4 EXTRUDERS."
```

Comment:
Can not compile Marlin with 
```
#define EXTRUDERS 3  //  or 5 
#define SWITCHING_EXTRUDER
```
because of error in **stepper.cpp**
> "'REV_E_DIR' was not declared in this scope"

```
  #if DISABLED(ADVANCE) && DISABLED(LIN_ADVANCE)
    if (motor_direction(E_AXIS)) {
      **REV_E_DIR();**
      count_direction[E_AXIS] = -1;
    }
    else {
      NORM_E_DIR();
      count_direction[E_AXIS] = 1;
    }
  #endif // !ADVANCE && !LIN_ADVANCE
```
and have no idea how to solve it now